### PR TITLE
I18n for UI strings

### PR DESCRIPTION
**Pull request by lbiedinger**
Use the Nuxt-i18n package for loading strings.

Deprecated Europeana specific string lookups for consistency and less complex test setup. 

Added English, German(partial) and Spanish(empty).

Moved all strings

### CODE REVIEW

File: plugins/europeana/search.js [hunk 1]
```diff
@@ -52,7 +54,7 @@ function display(field) {
 function fieldsForSearchResult(item) {
   let fields = {
     // TODO: fallback to description when API returns dcDescriptionLangAware
-    dcTitle: item.dcTitleLangAware ? display(item.dcTitleLangAware) : `No title provided for record ID ${item.id}`,
+    dcTitle: item.dcTitleLangAware ? display(item.dcTitleLangAware) : `${$t('messages.noTitle')}: ${item.id}`,
```
Comment (rwd) [inline plugins/europeana/search.js]: Use [named formatting](http://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) for the item ID.

File: components/search/SearchSelectedFacets.vue [hunk 2]
```diff
@@ -7,7 +7,7 @@
       class="mr-2"
       data-qa="filter badge"
     >
-      {{ selectedFacet.facetName | searchFacetHeader }}: {{ selectedFacet.fieldValue }}
+      {{ $t(`facets.${selectedFacet.facetName}`) }}: {{ selectedFacet.fieldValue }}
```
Comment (rwd) [inline components/search/SearchSelectedFacets.vue]: Use [named formatting](http://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting) with the colon and a placeholder for the facet field value included.

Comment (lbiedinger) [reply components/search/SearchSelectedFacets.vue]: that would make the translation unusable in the facet section headers, so would require unnecessary duplication of the same translation.

File: tests/unit/components/PageHeader.spec.js [hunk 3]
```diff
@@ -23,4 +43,12 @@ describe('components/search/PageHeader', () => {
     const logo = wrapper.find('[data-qa="logo"]');
     logo.attributes().src.should.match(/\/logo\..+\.svg$/);
   });
+
+  describe('when ENV enables the language selector', () => {
+    describe('it contains a language selector', () => {
```
Comment (rwd) [inline tests/unit/components/PageHeader.spec.js]: Let's also test the inverse, like:
```js
      const wrapper = factory();
      const selector = wrapper.findAll('[data-qa="language selector"]');
      selector.length.should.equal(0);
```

**Pull request merged**